fusor-ember-cli/app/routes/engine/discovered-host.js

```diff
@@ -11,10 +11,7 @@ export default Ember.Route.extend({
     var self = this;
     this.store.find('discovered-host').then(function(results) {
       controller.set('allDiscoveredHosts', results);
-      self.modelFor('deployment').get('discovered_hosts').then(function(results2) {
-        controller.set('selectedHypervisors', results2);
```

Comment:
I removed this, since `hypervisorModelIds` is calculated within the controller using a computed property